style.css

```diff
@@ -736,26 +736,27 @@ footer .submit:active {
 }
 
 .popUp.active {
+  position: fixed;
+  top: 10px;
+  height: 713px;
   display: flex;
   flex-direction: column;
-  width: 343px;
-  height: 713px;
-  position: absolute;
   background: #fff;
   border-radius: 16px;
   align-items: center;
-  padding: 16px;
-  gap: 12px;
+  padding: 16px 18px;
   font-family: 'Poppins', sans-serif;
+  justify-content: space-around;
+  z-index: 10;
 }
 
 .popUp .popHead {
   display: flex;
   flex-direction: column;
   align-items: flex-start;
-  gap: 12px;
   width: 311px;
-  height: 84px;
+  height: 100px;
+  margin-bottom: 12px;
 }
```

Comment:
- [ ] Kindly be noticed that you should make a scroll for the pop window project in the desktop size, to make the user see the description, the technologies, and the links for the source and live demo.

![Screenshot from 2022-07-27 00-59-22](https://user-images.githubusercontent.com/33752735/181127856-c3f58120-feca-40c8-9d40-927dd36c0d9d.png)

Reply:
@ShahierNashaat Dear please give me a few details about the changes u want from me
I am still confused by your request.

Reply:
I can't scroll down to see the description of the project please fix that